src/data.js

```diff
@@ -447,6 +447,23 @@ const pages = [
       'Node',
       'Vue'
     ],
+  },
+  {
+    name: 'Noman Gul',
+    description:
+      'Jr Software Engineer, Tutorial Maker, Biryani Lover',
+    url: 'https://howivscode.com/NomanGul',
```

Comment:
1. URL must ends with `/uses`
2. This list is not specific to vscode

Reply:
@armujahid oh, ok... Thanks :slightly_smiling_face: